modules/Microsoft.Network/vpnSites/readme.md

```diff
@@ -15,7 +15,7 @@ This module deploys a VPN Site.
 | :-- | :-- |
 | `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
 | `Microsoft.Authorization/roleAssignments` | [2020-10-01-preview](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2020-10-01-preview/roleAssignments) |
-| `Microsoft.Network/vpnSites` | [2021-05-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-05-01/vpnSites) |
+| `Microsoft.Network/vpnSites` | [2022-01-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/vpnSites) |
```

Comment:
Please update to latest version documented in ARR so that the link is mapped to the specific version used and not to generic latest (which can change over time)

Reply:
I mapped it to the latest offered by bicep. Which is the one we want to use?

Reply:
Same for all similar comments

Reply:
If we go with the latest from Bicep not yet documented by the Azure Resource Reference, we risk to not link the corresponding documentation. I'd vote for having the latest documented API, otherwise I don't see the point of linking the docs.

Reply:
For instance in this case the latest documented version is 2021-08-01 https://docs.microsoft.com/en-us/azure/templates/microsoft.network/2021-08-01/vpnsites?tabs=bicep

Reply:
I would also vote for what's in the documentation. That's the source of truth for customers anyways.

Reply:
OK fair enough. However, I'm not sure how this will scale over time. For each PR we'd need to check the Azure Resource Reference , which is definitely not up to date as you pointed out (sometimes for weeks). I wish there would be an API which is just consistent - but now we have Bicep with a certain set of versions, the API function in the tests that returns a different set of APIs and finally the Azure resource reference which introduces yet another that (but at least some documentation). Can't say I'm a big fan as it makes the update of our modules rather tedious

Reply:
Bare with me that it may take me some time to update my PR as I'm short on time until the end of the month 😕

Reply:
Updated VPN Site